documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -949,30 +949,30 @@ However, when it comes to link:http://community.jboss.org/wiki/HotRodProtocol[Ho
 ===== Data Consistency Problem
 Imagine you have two ATMs that connect using Hot Rod to a bank where an account's balance is stored.  Two closely followed operations to retrieve the latest balance could return 500 CHF (swiss francs) as shown below:
 
-image::images/server_modules_6.png[]
+image::images/server_modules_6.png[align="center", title="Concurrent readers"]
```

Comment:
@tristantarrant Image captions are currently to the bottom-left of each image, is this intended? It's a minor thing, but it does make the image captions appear asif they are headings (Figure x should give this away though)